docs/upgrade-guides/v0.10-->v1.0.md

```diff
@@ -0,0 +1,396 @@
+      unspent)
+
+Previously the response included a list of relative URLs pointed to
+transations' outputs:
+
+```json
+// Old
+[
+    "../transactions/2d431073e1477f3073a4693ac7ff9be5634751de1b8abaa1f4e19548ef0b4b0e/outputs/0"
+]
+
+// New
+[
+    {
+        "output": 0,
+        "transaction_id": "2d431073e1477f3073a4693ac7ff9be5634751de1b8abaa1f4e19548ef0b4b0e"
+    }
+]
+```
+
+In the future, we're planning to [upgrade this endpoint
+further](https://github.com/bigchaindb/bigchaindb/blob/99499b1f8783719a082813912ac9a0d363ae278f/bdb-ip.md#6-a-new-outputs-endpoint)
+to meet the requirements of [our
+users](https://github.com/bigchaindb/bigchaindb/issues/1227#issuecomment-307297473).
+
+
+### `GET /api/v1/statuses?tx_id`
+
+Documentation:
+
+- [Old](https://docs.bigchaindb.com/projects/server/en/v0.10.2/http-client-server-api.html#get--api-v1-statuses?tx_id=tx_id)
+- [New](https://docs.bigchaindb.com/projects/server/en/v1.0.0/http-client-server-api.html#get--api-v1-statuses?transaction_id=transaction_id)
+
+
+Changes:
+
+- All notion of `_links` was removed. In case of querying the status of a
+  transaction already included in a block marked `VALID`, no `_links` object is
+  provided anymore. The response object now only contains a single key value
+  pair named `status`
+- The query parameter `tx_id` was renamed to `transaction_id`, e.g. `GET
+  /api/v1/statuses?transaction_id=`
+
+
+```json
+// Old
+{
+    "status": "valid",
+        "_links": {
+            "tx": "/transactions/04c00267af82c161b4bf2ad4a47d1ddbfeb47eef1a14b8d51f37d6ee00ea5cdd"
+        }
+}
+
+// New
+{
+    "status": "valid",
+}
+```
+
+
+### `GET /api/v1/statuses?block_id`
+
+Documentation:
+
+- [Old](https://docs.bigchaindb.com/projects/server/en/v0.10.2/http-client-server-api.html#get--api-v1-statuses?block_id=block_id)
+- [New](https://docs.bigchaindb.com/projects/server/en/v1.0.0/http-client-server-api.html#get--api-v1-statuses?block_id=block_id)
+
+
+Changes:
+
+- All notion of `_links` was removed. The response object now only contains a
+  single key value pair named `status`
+
+
+```json
+// Old
+{
+    "status": "valid",
+        "_links": {
+            "tx": "/transactions/04c00267af82c161b4bf2ad4a47d1ddbfeb47eef1a14b8d51f37d6ee00ea5cdd"
+        }
+}
+
+// New
+{
+    "status": "valid",
+}
+```
+
+
+### `GET /api/v1/blocks?tx_id`
+
+Documentation:
+
+- [Old](https://docs.bigchaindb.com/projects/server/en/v0.10.2/http-client-server-api.html#get--api-v1-blocks?tx_id=tx_id&status=UNDECIDED|VALID|INVALID)
+- [New](https://docs.bigchaindb.com/projects/server/en/v1.0.0/http-client-server-api.html#get--api-v1-blocks?transaction_id=transaction_id&status=UNDECIDED|VALID|INVALID)
+
+
+Changes:
+
+- The query parameter `tx_id` was renamed to `transaction_id`, e.g. `GET
+  /api/v1/blocks?transaction_id`
+
+
+## Breaking Changes to the WebSocket Event Stream API
+
+In the event object sent to a listener, `tx_id` was renamed to
```

Comment:
What about the name of the WebSocket Event Stream API endpoint (was `/api/v1/streams/valid_tx`)? Is the `tx` now `transaction`?